main.go

```diff
@@ -20,6 +20,10 @@ import (
 	"flag"
 	"os"
 
+	"net/http"
+	"net/http/pprof"
+	_ "net/http/pprof" //nolint: all
```

Comment:
Why those 3 imports, the last one is not enough ?

Reply:
Oh yeah you're right my bad